libr/hash/hash.c

```diff
@@ -2,6 +2,13 @@
 
 #include <r_hash.h>
 #include "r_util.h"
+
+#if USE_LIB_XXHASH
```

Comment:
no need to ifdef for this. we add enough -I flags to make < and " includes work here

Reply:
If you treat them as external dependencies even though they are bundled, you'd just like to use `""`

Reply:
About this, I honestly don't know exactly how to do it correctly. Can you help me? The idea is that when we use sys-xxhash, we do want to use the system-provided header file. When we use embedded-xxhash, we do want to use our own header file. If we remove this `#if`, the same header fille will always be used.